examples/wasm_threads/README.md

```diff
@@ -0,0 +1,32 @@
+# Support workers with shared memory and one wasm binary
+
+This is a port of [wasm_threads `simple.rs` example](https://github.com/chemicstry/wasm_thread/tree/main?tab=readme-ov-file#simple).
+
+It should also work similarly with `wasm-bindgen-rayon` and `wasm-mt`.
+
+## Limitations
+
+It has a few considerable advantages over the `webworker*` examples, but also considerable disadvantages.
+
+For starters, `trunk serve` won't work currently, as it doesn't do Cross Site Isolation (setting 2 headers), which is [required for this approach to workers](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/SharedArrayBuffer#security_requirements).
```

Comment:
`trunk serve` can set arbitrary headers now IIRC. That's missing in the examples right now. I'll add that.

Reply:
Added them, further improved documentation.